tests/test_kernel_magics.py

```diff
@@ -5,6 +5,7 @@
 import remotespark.utils.configuration as conf
 from remotespark.kernels.kernelmagics import KernelMagics
 from remotespark.utils.constants import Constants
+from remotespark.livyclientlib.livyclienttimeouterror import LivyClientTimeoutError
```

Comment:
Re-arrange alphabetically. They should be:
 from remotespark.kernels.kernelmagics import KernelMagics
 from remotespark.livyclientlib.livyclienttimeouterror import LivyClientTimeoutError
 from remotespark.utils.constants import Constants
import remotespark.utils.configuration as conf